src/fixit/api.py

```diff
@@ -186,12 +199,25 @@ def fixit_file(
 
 
 def _fixit_file_wrapper(
-    path: Path, *, autofix: bool = False, options: Optional[Options] = None
+    path: Path,
+    *,
+    autofix: bool = False,
+    options: Optional[Options] = None,
+    trees: Optional[Dict[Path, ConfigTree]] = None,
```

Comment:
TODO: initialise the process pool with the config trees rather than serializing them into each task.